doc/src/explanation/active-deprecations.md

```diff
@@ -433,8 +433,14 @@ than an element method.
 ### Cleaning up symbolic version functions in ``ntheory``
 
 Symbolic functions in ``ntheory`` have been moved to ``functions``.
-Specifically, the following functions.
-
+All of these functions can be imported like ``from sympy import mobius`` which
+continues to work just fine after these changes as well as it did before.
+However, if you are using ``from sympy.ntheory import mobius``, you need to
+change to ``from sympy import mobius``. Specifically, the following functions.
+
+* ``sympy.ntheory.factor_.primenu``
+* ``sympy.ntheory.factor_.primeomega``
+* ``sympy.ntheory.factor_.totient``
 * ``sympy.ntheory.residue_ntheory.mobius``
```

Comment:
Okay here is how I suggest to word the whole thing:
```
### Relocate symbolic functions from ``ntheory`` to ``functions``

The following symbolic functions in ``ntheory`` have been moved to
``functions``:

* ``sympy.ntheory.factor_.primenu``
* ``sympy.ntheory.factor_.primeomega``
* ``sympy.ntheory.factor_.totient``
* ``sympy.ntheory.residue_ntheory.mobius``

Code that imports these functions from top-level like ``from sympy import
mobius`` will continue to work fine. However code that imports these from the
fully qualified module like ``from sympy.ntheory import mobius`` or ``from
sympy.ntheory.residue_ntheory import mobius`` will now see a deprecation
warning. The new location for these functions is in ``sympy.functions`` but the
intended way to import them is still from top-level like ``from sympy import
mobius``.
```

Reply:
Thank you.